problem_builder/answer.py

```diff
@@ -53,6 +52,11 @@ class AnswerMixin(XBlockWithPreviewMixin, XBlockWithTranslationServiceMixin, Stu
     """
     Mixin to give an XBlock the ability to read/write data to the Answers DB table.
     """
+    USER_STATE_FIELDS = ['edxnotes_visibility', ]
```

Comment:
What is this `edxnotes_visibility` field, and why do we care about it?

Reply:
@bradenmacdonald From the attr definition:
> Indicates whether Student Notes are visible in the course. Students can also show or hide their notes in the courseware.

It appeared everywhere before I started working on this PR. Kept it for compatibility reasons, but will remove.